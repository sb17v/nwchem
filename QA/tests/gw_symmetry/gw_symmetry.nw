title "GW Methane Td"
start
echo

memory 2000 mb

 geometry units au
   c   0.0000000      0.0000000      0.0000000
   h   1.1828637      1.1828637      1.1828637
   symmetry group Td
 end

basis "ao basis" bse spherical
 * library cc-pvtz
end

basis "ri basis" bse spherical
 * library cc-pvtz-rifit
end


dft
 direct
 grid nodisk
 tolerances acccoul 12
 noprint "final vectors analysis"
 xc pbe96
end

gw
 states alpha occ 1
end

task dft gw


dft
 sym off
end

task dft gw

