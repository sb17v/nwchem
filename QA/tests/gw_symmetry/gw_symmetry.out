 argument  1 = gw_symmetry.nw
  NWChem w/ OpenMP: maximum threads =    8



============================== echo of input deck ==============================
title "GW Methane Td"
start
echo

memory 2000 mb

 geometry units au
   c   0.0000000      0.0000000      0.0000000
   h   1.1828637      1.1828637      1.1828637
   symmetry group Td
 end

basis "ao basis" bse spherical
 * library cc-pvtz
end

basis "ri basis" bse spherical
 * library cc-pvtz-rifit
end


dft
 direct
 grid nodisk
 tolerances acccoul 12
 noprint "final vectors analysis"
 xc pbe96
end

gw
 states alpha occ 1
end

task dft gw


dft
 sym off
end

task dft gw

================================================================================


                                         
                                         


             Northwest Computational Chemistry Package (NWChem) 7.2.0
             --------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2022
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = WE41476
    program         = nwchem
    date            = Wed Jul 12 13:50:26 2023

    compiled        = Wed_Jul_12_13:50:03_2023
    source          = /Users/meji656/Sources/nwchem_bse
    nwchem branch   = 7.2.0
    nwchem revision = nwchem_on_git-4850-g011c901253
    ga revision     = 5.8.0
    use scalapack   = T
    input           = gw_symmetry.nw
    prefix          = gw_symmetry.
    data base       = ./gw_symmetry.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   65535996 doubles =    500.0 Mbytes
    stack    =   65536001 doubles =    500.0 Mbytes
    global   =  131072000 doubles =   1000.0 Mbytes (distinct from heap & stack)
    total    =  262143997 doubles =   2000.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                   GW Methane Td
                                   -------------
 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------
  Looking for out-of-plane bends
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 c                    6.0000     0.00000000     0.00000000     0.00000000
    2 h                    1.0000     1.18286370     1.18286370     1.18286370
    3 h                    1.0000    -1.18286370    -1.18286370     1.18286370
    4 h                    1.0000    -1.18286370     1.18286370    -1.18286370
    5 h                    1.0000     1.18286370    -1.18286370    -1.18286370

      Atomic Mass 
      ----------- 

      c                 12.000000
      h                  1.007825


 Effective nuclear repulsion energy (a.u.)      13.5076651723

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name              Td       
 Group number             42
 Group order              24
 No. of unique centers     2

      Symmetry unique atoms

     1    2



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.08417
    2 Stretch                  1     3                       1.08417
    3 Stretch                  1     4                       1.08417
    4 Stretch                  1     5                       1.08417
    5 Bend                     2     1     3               109.47122
    6 Bend                     2     1     4               109.47122
    7 Bend                     2     1     5               109.47122
    8 Bend                     3     1     4               109.47122
    9 Bend                     3     1     5               109.47122
   10 Bend                     4     1     5               109.47122


            XYZ format geometry
            -------------------
     5
 geometry
 c                     0.00000000     0.00000000     0.00000000
 h                     0.62594456     0.62594456     0.62594456
 h                    -0.62594456    -0.62594456     0.62594456
 h                    -0.62594456     0.62594456    -0.62594456
 h                     0.62594456    -0.62594456    -0.62594456

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 h                |   1 c                |     2.04878  |     2.04878
    3 h                |   1 c                |     2.04878  |     2.04878
    4 h                |   1 c                |     2.04878  |     2.04878
    5 h                |   1 c                |     2.04878  |     2.04878
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          4
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 h                |   1 c                |   3 h                |   109.47
    2 h                |   1 c                |   4 h                |   109.47
    2 h                |   1 c                |   5 h                |   109.47
    3 h                |   1 c                |   4 h                |   109.47
    3 h                |   1 c                |   5 h                |   109.47
    4 h                |   1 c                |   5 h                |   109.47
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          6
 ==============================================================================





 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvtz                   on all atoms 




 Summary of "ri basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                       cc-pvtz-rifit                on all atoms 



                                 NWChem GW Module
                                 ----------------


                                   GW Methane Td



                                 NWChem DFT Module
                                 -----------------


                                   GW Methane Td


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  c (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  8.23600000E+03  0.000531
  1 S  1.23500000E+03  0.004108
  1 S  2.80800000E+02  0.021087
  1 S  7.92700000E+01  0.081853
  1 S  2.55900000E+01  0.234817
  1 S  8.99700000E+00  0.434401
  1 S  3.31900000E+00  0.346129
  1 S  3.64300000E-01 -0.008983

  2 S  9.05900000E-01  1.000000

  3 S  8.23600000E+03 -0.000113
  3 S  1.23500000E+03 -0.000878
  3 S  2.80800000E+02 -0.004540
  3 S  7.92700000E+01 -0.018133
  3 S  2.55900000E+01 -0.055760
  3 S  8.99700000E+00 -0.126895
  3 S  3.31900000E+00 -0.170352
  3 S  3.64300000E-01  0.598684

  4 S  1.28500000E-01  1.000000

  5 P  1.87100000E+01  0.014031
  5 P  4.13300000E+00  0.086866
  5 P  1.20000000E+00  0.290216

  6 P  3.82700000E-01  1.000000

  7 P  1.20900000E-01  1.000000

  8 D  1.09700000E+00  1.000000

  9 D  3.18000000E-01  1.000000

 10 F  7.61000000E-01  1.000000

  h (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.38700000E+01  0.006068
  1 S  5.09500000E+00  0.045308
  1 S  1.15900000E+00  0.202822

  2 S  3.25800000E-01  1.000000

  3 S  1.02700000E-01  1.000000

  4 P  1.40700000E+00  1.000000

  5 P  3.88000000E-01  1.000000

  6 D  1.05700000E+00  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 c                          cc-pvtz                 10       30   4s3p2d1f
 h                          cc-pvtz                  6       14   3s2p1d




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 c                          cc-pvtz                 10       30   4s3p2d1f
 h                          cc-pvtz                  6       14   3s2p1d


      Symmetry analysis of basis
      --------------------------

        a1         11
        a2          0
        e          12
        t1         15
        t2         48

  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     5
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  50
          This is a Direct SCF calculation.
          AO basis - number of functions:    86
                     number of shells:    34
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                        PBE96 Method XC Functional
          PerdewBurkeErnzerhof Exchange Functional  1.000          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          c                   0.70       49           6.0       434
          h                   0.35       45           7.0       434
          Grid pruning is: on 
          Number of quadrature shells:    94
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         50 iters            50 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-12


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -39.66973321

      Non-variational initial energy
      ------------------------------

 Total energy =     -40.267243
 1-e energy   =     -78.381769
 2-e energy   =      24.606861
 HOMO         =      -0.524977
 LUMO         =       0.083171


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1          2 a2          3 e           4 t1          5 t2      

  Orbital symmetries:

     1 a1          2 a1          3 t2          4 t2          5 t2      
     6 a1          7 t2          8 t2          9 t2         10 t2      
    11 t2         12 t2         13 a1         14 t2         15 t2      

  WARNING: movecs_in_org=atomic not equal to movecs_in=./gw_symmetry.movecs
   Time after variat. SCF:      0.3
   Time prior to 1st pass:      0.3

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       65.53            65532636
          Stack Space remaining (MW):       65.54            65535396

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -40.4526766257 -5.40D+01  2.73D-03  1.58D-01     0.6
 d= 0,ls=0.0,diis     2    -40.4605212533 -7.84D-03  1.18D-03  2.13D-02     0.8
 d= 0,ls=0.0,diis     3    -40.4604458885  7.54D-05  6.05D-04  2.16D-02     1.1
 d= 0,ls=0.0,diis     4    -40.4630635417 -2.62D-03  2.24D-05  6.66D-06     1.3
 d= 0,ls=0.0,diis     5    -40.4630646869 -1.15D-06  2.13D-06  1.39D-07     1.6
 d= 0,ls=0.0,diis     6    -40.4630647071 -2.03D-08  7.01D-08  1.45D-10     1.9


         Total DFT energy =      -40.463064707119
      One electron energy =      -79.987919842739
           Coulomb energy =       32.854590174254
    Exchange-Corr. energy =       -6.837400210970
 Nuclear repulsion energy =       13.507665172336

 Numeric. integr. density =       10.000008758007

     Total iterative time =      1.5s



                  Occupations of the irreducible representations
                  ----------------------------------------------

                     irrep           alpha         beta
                     --------     --------     --------
                     a1                2.0          2.0
                     a2                0.0          0.0
                     e                 0.0          0.0
                     t1                0.0          0.0
                     t2                3.0          3.0


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          11.280920087173           0.000000000000           0.000000000000
           0.000000000000          11.280920087173           0.000000000000
           0.000000000000           0.000000000000          11.280920087173

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0     -0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0     -0.000000     -0.000000     -0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1     -0.000000     -0.000000     -0.000000      0.000000

     2   2 0 0     -6.283208     -5.939937     -5.939937      5.596666
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -6.283208     -5.939937     -5.939937      5.596666
     2   0 1 1     -0.000000     -0.000000     -0.000000      0.000000
     2   0 0 2     -6.283208     -5.939937     -5.939937      5.596666

  
                             ***********************
                             ***        GW       ***
                             ***********************
  
                      Basis "ri basis" -> "ri basis" (spherical)
                      -----
  c (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  2.80556612E+02  1.000000

  2 S  6.02302472E+01  1.000000

  3 S  2.21166508E+01  1.000000

  4 S  6.86209329E+00  1.000000

  5 S  1.56225495E+00  1.000000

  6 S  9.12450455E-01  1.000000

  7 S  5.02749170E-01  1.000000

  8 S  2.15823487E-01  1.000000

  9 P  2.42318652E+01  1.000000

 10 P  6.47645142E+00  1.000000

 11 P  2.12908847E+00  1.000000

 12 P  1.03756397E+00  1.000000

 13 P  5.49412750E-01  1.000000

 14 P  2.81526522E-01  1.000000

 15 D  9.71898108E+00  1.000000

 16 D  3.03514750E+00  1.000000

 17 D  1.46777376E+00  1.000000

 18 D  5.73708279E-01  1.000000

 19 D  2.90555897E-01  1.000000

 20 F  2.27562013E+00  1.000000

 21 F  1.19688989E+00  1.000000

 22 F  5.41574872E-01  1.000000

 23 G  1.21661016E+00  1.000000

  h (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  8.51282759E+00  1.000000

  2 S  1.87308912E+00  1.000000

  3 S  5.26184269E-01  1.000000

  4 S  2.89739899E-01  1.000000

  5 P  2.37254916E+00  1.000000

  6 P  1.18040845E+00  1.000000

  7 P  6.03822042E-01  1.000000

  8 D  1.80963732E+00  1.000000

  9 D  1.14397261E+00  1.000000

 10 F  1.80630606E+00  1.000000



 Summary of "ri basis" -> "ri basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 c                       cc-pvtz-rifit              23       81   8s6p5d3f1g
 h                       cc-pvtz-rifit              10       30   4s3p2d1f



                 Memory Information
                 ------------------
          Available GA per process is           999.9 MBs
          Available stack per process is        500.0 MBs
          Available heap per process is         499.9 MBs

          Peak memory in Initialization
              Peak GA per process is                  1.1 MBs
              Peak stack per process is               0.1 MBs
              Peak heap per process is                1.2 MBs
              Memory not in MA per process            0.0 MBs

          Peak memory in AFF Calculation
              Peak GA per process is                  2.5 MBs
              Peak stack per process is               1.3 MBs
              Peak heap per process is                0.0 MBs
              Memory not in MA per process            1.3 MBs

          Computing 2-center integrals
          Computing Inverse Cholesky factor
          Computing 3-center integrals
   Transform ERIs
   Distribute ERIs
   Read inverse
   Orthogonalize ERIs
          Computing V_xc ... 
          Computing Sigma_x
          Computing RPA polarizability
          Solving quasiparticle equations

                  Alpha Orbitals
          State     Energy (eV)     Error (eV)
          -----------------------------------
@GW         5        -13.866           0.000

   Applying  -4.4168 eV scissor-shift to rest of particle states


     GW Timing Statistics
   ------------------------
   Init.   :          0.1s
     ERIs  :          0.3s
     Distr.:          0.0s
     Vxc   :          0.2s
   Sigma_x :          0.0s
   RPA     :          0.0s
   Wmn     :          0.0s
   Sigma_c :          0.0s
   GW total:          0.5s


 Task  times  cpu:        2.4s     wall:        2.4s


                                NWChem Input Module
                                -------------------



                                 NWChem GW Module
                                 ----------------


                                   GW Methane Td



                                 NWChem DFT Module
                                 -----------------


                                   GW Methane Td




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 c                          cc-pvtz                 10       30   4s3p2d1f
 h                          cc-pvtz                  6       14   3s2p1d


      Symmetry analysis of basis
      --------------------------

        a1         11
        a2          0
        e          12
        t1         15
        t2         48

  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     5
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: on 
          Maximum number of iterations:  50
          This is a Direct SCF calculation.
          AO basis - number of functions:    86
                     number of shells:    34
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                        PBE96 Method XC Functional
          PerdewBurkeErnzerhof Exchange Functional  1.000          
            Perdew 1991 LDA Correlation Functional  1.000 local    
           PerdewBurkeErnz. Correlation Functional  1.000 non-local

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          c                   0.70       49           6.0       434
          h                   0.35       45           7.0       434
          Grid pruning is: on 
          Number of quadrature shells:   229
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         50 iters            50 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-12


 Loading old vectors from job with title :

GW Methane Td


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 a1          2 a2          3 e           4 t1          5 t2      

  Orbital symmetries:

     1 a1          2 a1          3 t2          4 t2          5 t2      
     6 a1          7 t2          8 t2          9 t2         10 t2      
    11 t2         12 t2         13 t2         14 t2         15 t2      

   Time after variat. SCF:      2.5
   Time prior to 1st pass:      2.5

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       65.53            65532636
          Stack Space remaining (MW):       65.54            65535396

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1    -40.4630647077 -5.40D+01  1.34D-08  1.68D-12     3.5
 d= 0,ls=0.0,diis     2    -40.4630647077 -2.15D-12  7.98D-09  3.73D-12     4.4


         Total DFT energy =      -40.463064707739
      One electron energy =      -79.987938823681
           Coulomb energy =       32.854611046058
    Exchange-Corr. energy =       -6.837402102452
 Nuclear repulsion energy =       13.507665172336

 Numeric. integr. density =       10.000008760592

     Total iterative time =      1.9s



                  Occupations of the irreducible representations
                  ----------------------------------------------

                     irrep           alpha         beta
                     --------     --------     --------
                     a1                2.0          2.0
                     a2                0.0          0.0
                     e                 0.0          0.0
                     t1                0.0          0.0
                     t2                3.0          3.0


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          11.280920087173           0.000000000000           0.000000000000
           0.000000000000          11.280920087173           0.000000000000
           0.000000000000           0.000000000000          11.280920087173

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0     -0.000000     -5.000000     -5.000000     10.000000

     1   1 0 0     -0.000000     -0.000000     -0.000000      0.000000
     1   0 1 0     -0.000000     -0.000000     -0.000000      0.000000
     1   0 0 1     -0.000000     -0.000000     -0.000000      0.000000

     2   2 0 0     -6.283210     -5.939938     -5.939938      5.596666
     2   1 1 0     -0.000000     -0.000000     -0.000000      0.000000
     2   1 0 1     -0.000000     -0.000000     -0.000000      0.000000
     2   0 2 0     -6.283210     -5.939938     -5.939938      5.596666
     2   0 1 1     -0.000000     -0.000000     -0.000000      0.000000
     2   0 0 2     -6.283210     -5.939938     -5.939938      5.596666

  
                             ***********************
                             ***        GW       ***
                             ***********************
  

                 Memory Information
                 ------------------
          Available GA per process is           999.9 MBs
          Available stack per process is        500.0 MBs
          Available heap per process is         499.9 MBs

          Peak memory in Initialization
              Peak GA per process is                  1.1 MBs
              Peak stack per process is               0.1 MBs
              Peak heap per process is                1.2 MBs
              Memory not in MA per process            0.0 MBs

          Peak memory in AFF Calculation
              Peak GA per process is                  2.5 MBs
              Peak stack per process is               1.3 MBs
              Peak heap per process is                0.0 MBs
              Memory not in MA per process            1.3 MBs

          Computing 2-center integrals
          Computing Inverse Cholesky factor
          Computing 3-center integrals
   Transform ERIs
   Distribute ERIs
   Read inverse
   Orthogonalize ERIs
          Computing V_xc ... 
          Computing Sigma_x
          Computing RPA polarizability
          Solving quasiparticle equations

                  Alpha Orbitals
          State     Energy (eV)     Error (eV)
          -----------------------------------
@GW         5        -13.866           0.000

   Applying  -4.4168 eV scissor-shift to rest of particle states


     GW Timing Statistics
   ------------------------
   Init.   :          0.0s
     ERIs  :          0.2s
     Distr.:          0.0s
     Vxc   :          0.5s
   Sigma_x :          0.0s
   RPA     :          0.0s
   Wmn     :          0.0s
   Sigma_c :          0.0s
   GW total:          0.8s


 Task  times  cpu:        2.8s     wall:        2.8s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays


MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        29	        62
	current total bytes		         0	         0
	maximum total bytes		  32636240	  32078504
	maximum total K-bytes		     32637	     32079
	maximum total M-bytes		        33	        33


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, Y. Alexeev, J. Anchell,
       V. Anisimov, F. W. Aquino, R. Atta-Fynn, J. Autschbach, N. P. Bauman,
     J. C. Becca, D. E. Bernholdt, K. Bhaskaran-Nair, S. Bogatko, P. Borowski,
         J. Boschen, J. Brabec, A. Bruner, E. Cauet, Y. Chen, G. N. Chuev,
      C. J. Cramer, J. Daily, M. J. O. Deegan, T. H. Dunning Jr., M. Dupuis,
   K. G. Dyall, G. I. Fann, S. A. Fischer, A. Fonari, H. Fruchtl, L. Gagliardi,
      J. Garza, N. Gawande, S. Ghosh, K. Glaesemann, A. W. Gotz, J. Hammond,
       V. Helms, E. D. Hermes, K. Hirao, S. Hirata, M. Jacquelin, L. Jensen,
   B. G. Johnson, H. Jonsson, R. A. Kendall, M. Klemm, R. Kobayashi, V. Konkov,
      S. Krishnamoorthy, M. Krishnan, Z. Lin, R. D. Lins, R. J. Littlefield,
      A. J. Logsdail, K. Lopata, W. Ma, A. V. Marenich, J. Martin del Campo,
   D. Mejia-Rodriguez, J. E. Moore, J. M. Mullin, T. Nakajima, D. R. Nascimento,
    J. A. Nichols, P. J. Nichols, J. Nieplocha, A. Otero-de-la-Roza, B. Palmer,
    A. Panyala, T. Pirojsirikul, B. Peng, R. Peverati, J. Pittner, L. Pollack,
   R. M. Richard, P. Sadayappan, G. C. Schatz, W. A. Shelton, D. W. Silverstein,
   D. M. A. Smith, T. A. Soares, D. Song, M. Swart, H. L. Taylor, G. S. Thomas,
            V. Tipparaju, D. G. Truhlar, K. Tsemekhman, T. Van Voorhis,
      A. Vazquez-Mayagoitia, P. Verma, O. Villa, A. Vishnu, K. D. Vogiatzis,
        D. Wang, J. H. Weare, M. J. Williamson, T. L. Windus, K. Wolinski,
        A. T. Wong, Q. Wu, C. Yang, Q. Yu, M. Zacharias, Z. Zhang, Y. Zhao,
                                and R. J. Harrison
                        "NWChem: Past, present, and future
                         J. Chem. Phys. 152, 184102 (2020)
                               doi:10.1063/5.0004997

                                      AUTHORS
                                      -------
  E. Apra, E. J. Bylaska, N. Govind, K. Kowalski, M. Valiev, D. Mejia-Rodriguez,
       A. Kunitsa, N. P. Bauman, A. Panyala, W. A. de Jong, T. P. Straatsma,
   H. J. J. van Dam, D. Wang, T. L. Windus, J. Hammond, J. Autschbach, A. Woods,
    K. Bhaskaran-Nair, J. Brabec, K. Lopata, S. A. Fischer, S. Krishnamoorthy,
     M. Jacquelin, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov, F. Aquino,
     S. Hirata, M. T. Hackler, E. Hermes, L. Jensen, J. E. Moore, J. C. Becca,
      V. Konjkov, T. Risthaus, M. Malagoli, A. Marenich, A. Otero-de-la-Roza,
        J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan,
        A. Fonari, M. J. Williamson, R. J. Harrison, J. R. Rehr, M. Dupuis,
     D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan,
     B. E. Van Kuiken, A. Vazquez-Mayagoitia, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        5.2s     wall:        5.2s
